extensions/wrapper/wrapper-common-mappers/src/test/java/de/sovity/edc/ext/wrapper/api/common/mappers/policy/AtomicConstraintMapperTest.java

```diff
@@ -1,7 +1,6 @@
-package de.sovity.edc.ext.wrapper.api.common.mappers.utils;
+package de.sovity.edc.ext.wrapper.api.common.mappers.policy;
```

Comment:
Missing copyright header
```suggestion
/*
 *  Copyright (c) 2022 sovity GmbH
 *
 *  This program and the accompanying materials are made available under the
 *  terms of the Apache License, Version 2.0 which is available at
 *  https://www.apache.org/licenses/LICENSE-2.0
 *
 *  SPDX-License-Identifier: Apache-2.0
 *
 *  Contributors:
 *       sovity GmbH - initial API and implementation
 *
 */

package de.sovity.edc.ext.wrapper.api.common.mappers.policy;
```